js/views-manager.js

```diff
@@ -11,6 +11,19 @@ var ViewsManager = {
   projects: [],
   $controlsContainer: $("#project-container .controls"),
   $projectsContainer: $("#project-container .projects"),
+  NotFoundView: {
```

Comment:
Since you use this for:
* nothing found in search,
* broken direct link, and
* prompt for saving favourites

Maybe we should name this more generically than "NotFoundView"? Something like NotificationView or MessageView